nuxeo-features/nuxeo-platform-notification/nuxeo-platform-notification-core/src/test/java/org/nuxeo/ecm/platform/ec/notification/TestRenderingService.java

```diff
@@ -0,0 +1,70 @@
+/*
+ * (C) Copyright 2007-2019 Nuxeo (http://nuxeo.com/) and others.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ * Contributors:
+ *     Thierry Martins <tmartins@nuxeo.com>
+ */
+package org.nuxeo.ecm.platform.ec.notification;
+
+import static org.junit.Assert.assertTrue;
+
+import java.util.Collection;
+
+import javax.inject.Inject;
+
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.nuxeo.ecm.core.api.CoreSession;
+import org.nuxeo.ecm.core.api.DocumentModel;
+import org.nuxeo.ecm.core.test.CoreFeature;
+import org.nuxeo.ecm.platform.ec.notification.email.NotificationsRenderingEngine;
+import org.nuxeo.ecm.platform.rendering.RenderingResult;
+import org.nuxeo.ecm.platform.rendering.RenderingService;
+import org.nuxeo.ecm.platform.rendering.impl.DocumentRenderingContext;
+import org.nuxeo.runtime.api.Framework;
+import org.nuxeo.runtime.test.runner.Deploy;
+import org.nuxeo.runtime.test.runner.Features;
+import org.nuxeo.runtime.test.runner.FeaturesRunner;
+
+@RunWith(FeaturesRunner.class)
+@Features(CoreFeature.class)
+@Deploy("org.nuxeo.ecm.platform.notification.core")
+@Deploy("org.nuxeo.ecm.platform.notification.core.tests:notification-contrib.xml")
+public class TestRenderingService {
+
+    private static final String MYDESCRIPTION = "mydescription";
```

Comment:
i think can be replaced by `MY_DESCRIPTION `

Reply:
yes you're right, but again I want to keep the same code between master and 10.10